src/OVAL/probes/probe/main.c

```diff
@@ -257,6 +258,7 @@ int main(int argc, char *argv[])
 			if (chdir(rootdir) != 0) {
 				fail(errno, "chdir", __LINE__ -1);
 			}
+			dlopen("libgcc_s.so.1", RTLD_NOW | RTLD_GLOBAL);
```

Comment:
It is quite complicated scenario.
I was trying to find some other function to force to load right library before chroot, but without success.

It fail during pthread_cancel() I don't know clean way how to find what does the function need.
And we can have similar functions.

Code compiled using clang gives me same error ```libgcc_s.so.1 must be installed for pthread_cancel to work
```
But after fix it isn't able to find dlopen.

@mpreisler What do you recommend to set during configure?